third_party/mkl-dnn.BUILD

```diff
@@ -49,47 +50,50 @@ _DNNL_RUNTIME_OMP = {
 }
 
 template_rule(
-    name = "third_party/oneDNN/include_dnnl_version",
-    src = "third_party/oneDNN/include/oneapi/dnnl/dnnl_version.h.in",
-    out = "third_party/oneDNN/include/oneapi/dnnl/dnnl_version.h",
+    name = "include_dnnl_version",
+    src = "include/oneapi/dnnl/dnnl_version.h.in",
+    out = "include/oneapi/dnnl/dnnl_version.h",
     substitutions = {
-        "@DNNL_VERSION_MAJOR@": "2",
-        "@DNNL_VERSION_MINOR@": "7",
-        "@DNNL_VERSION_PATCH@": "3",
-        "@DNNL_VERSION_HASH@": "7710bdc92064a08b985c5cbdb09de773b19cba1f",
+        "@DNNL_VERSION_MAJOR@": "3",
+        "@DNNL_VERSION_MINOR@": "1",
+        "@DNNL_VERSION_PATCH@": "0",
+        "@DNNL_VERSION_HASH@": "928065bd62372d2824900a86c5438bba407c98d1",
     },
 )
 
 template_rule(
-    name = "third_party/oneDNN/include_dnnl_config",
-    src = "third_party/oneDNN/include/oneapi/dnnl/dnnl_config.h.in",
-    out = "third_party/oneDNN/include/oneapi/dnnl/dnnl_config.h",
+    name = "include_dnnl_config",
+    src = "include/oneapi/dnnl/dnnl_config.h.in",
+    out = "include/oneapi/dnnl/dnnl_config.h",
     substitutions = _DNNL_RUNTIME_OMP,
 )
 
 cc_library(
     name = "mkl-dnn",
     srcs = glob([
-        "third_party/oneDNN/src/common/*.cpp",
-        "third_party/oneDNN/src/cpu/**/*.cpp",
+        "src/common/*.cpp",
+        "src/cpu/**/*.cpp",
     ], exclude=[
-        "third_party/oneDNN/src/cpu/aarch64/**/*.cpp",
+        "src/cpu/aarch64/**/*.cpp",
+        "src/cpu/rv64/**/*.cpp",
     ]),
     hdrs = glob([
-        "third_party/oneDNN/include/oneapi/dnnl/*.h",
-        "third_party/oneDNN/include/oneapi/dnnl/*.hpp",
-        "third_party/oneDNN/include/*.h",
-        "third_party/oneDNN/include/*.hpp",
-        "third_party/oneDNN/src/cpu/**/*.hpp",
-        "third_party/oneDNN/src/cpu/**/*.h",
-        "third_party/oneDNN/src/common/*.hpp",
-        "third_party/oneDNN/src/common/ittnotify/jitprofiling.h",
+        "include/oneapi/dnnl/*.h",
+        "include/oneapi/dnnl/*.hpp",
+        "include/*.h",
+        "include/*.hpp",
+        "src/cpu/**/*.hpp",
+        "src/cpu/**/*.h",
+        "src/common/*.hpp",
+        "src/common/ittnotify/jitprofiling.h",
     ], exclude=[
-        "third_party/oneDNN/src/cpu/aarch64/**/*.hpp",
-        "third_party/oneDNN/src/cpu/aarch64/**/*.h",
+        "src/cpu/aarch64/**/*.hpp",
+        "src/cpu/aarch64/**/*.h",
+        "src/cpu/rv64/**/*.hpp",
+        "src/cpu/rv64/**/*.h",
```

Comment:
Hi @jgong5 We excluded RV64 files due to CI failures (build issue). Do you think it's OK?

Reply:
Let's ignore these newly added RISCV files.

Reply:
OK